DESCRIPTION

```diff
@@ -69,6 +69,9 @@ Suggests:
     survAUC,
     svglite,
     xgboost
+Remotes:
+    mlr-org/mlr3tuning,
+    mlr-org/paradox
```

Comment:
Is this still required? IIRC we agreed on not using any dev versions in the book.

Reply:
We just do this inside a PR when the change is not on CRAN yet.